setup.py

```diff
@@ -56,7 +56,7 @@
             "pytest",
             "pytest-cov",
             "pytest-mock",
-            "substratools>=0.21.0",
+            "substratools==0.21.1",
```

Comment:
Why are we suddently pinning `substratools` version?

Reply:
Fixed